### PR TITLE
Clear some cut-and-paste cruft out of our Gradle files.

### PR DESCRIPTION
**Pull request by chrisr3**
The experimental modules are part of the main project and so inherit repositories, plugins and properties.

### CODE REVIEW

File: experimental/avalanche/build.gradle [hunk 1]
```diff
@@ -1,41 +1,15 @@
...
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.3'
-    }
-}
-
 plugins {
     id "org.jetbrains.kotlin.jvm"
-    id 'com.github.johnrengelman.shadow' version '2.0.3'
-    id 'java'
+    id 'com.github.johnrengelman.shadow' version '2.0.4'
```
Comment (Clintonio) [inline experimental/avalanche/build.gradle]: I think we're using this else where right? If we are could you standardise it in the ext part of the root build.gradle?

File: experimental/avalanche/build.gradle [hunk 2]
```diff
@@ -1,41 +1,15 @@
...
-    kotlinOptions {
-        jvmTarget = "1.8"
-    }
-}
-compileTestKotlin {
-    kotlinOptions {
-        jvmTarget = "1.8"
-    }
+    compile "info.picocli:picocli:3.0.1"
+    testCompile group: 'junit', name: 'junit', version: junit_version
```
Comment (Clintonio) [inline experimental/avalanche/build.gradle]: We should standardise on one style - string or map.

**Comment by Clintonio** [comment]
Also call it what it is - cargo culting.

File: experimental/avalanche/build.gradle (hunk 1 above)
Comment (chrisr3) [reply experimental/avalanche/build.gradle]: It's there already - but this is the Plugins DSL and I cannot use it here :man_facepalming:

File: experimental/avalanche/build.gradle (hunk 2 above)
Comment (chrisr3) [reply experimental/avalanche/build.gradle]: I'll standardise on "string" here.

File: experimental/avalanche/build.gradle (hunk 1 above)
Comment (Clintonio) [reply experimental/avalanche/build.gradle]: Can we just kill the DSL? It's sub-standard for us right now?

**Pull request merged**